CONTRIBUTING.md

```diff
@@ -250,7 +250,7 @@ Please ensure that your Javadoc comments are correct. Eclipse can check this for
 	- Set the context to `/iaf-example` (or `/iaf-test`, for running tests from the project ibis-adapterframework-test).
 	- Add `-Ddtap.stage=LOC` to VM Options.
     - In the "On Update" section, select "Update Classes and Resources" so that classes can be automatically updated and reloaded after project build (providing this is supported by your JDK)
-    - Under the section "Build", add a build step to build the console-frontend project via Maven. This should be a Maven build action, running target `install` in the project "Frank Console Frontend".
+    - Under the section "Before launch", add a build step to build the console-frontend project via Maven. Add a Maven Goal action, running command `install` in the project "ibis-adapterframework-console-frontend".
```

Comment:
De officiele naam is 'Frank Console Frontend', de artifact naam is 'ibis-adapterframework-console-frontend'. Ik weet niet of jullie de projecten anders hebben geimporteerd, maar het is denk ik wel handig de module naam te gebruiken en niet de naam van de jar die er uit komt.

Reply:
In dit menu gebruikt hij niet de naam maar de artifact naam, voor de duidelijkheid heb ik het daarom aangepast
![image](https://user-images.githubusercontent.com/4380412/236422529-bed297e8-fe98-4681-bfdc-73fa2081258a.png)

Reply:
Dan heb je de artifacts en niet de project files geïmporteerd?

Reply:
Dit is hoe Intellij zelf alles heeft geimporteerd vanuit Maven, hij ziet ook wel alle projecten en niet alleen de artifacts maar toch is dit hoe het is aangegeven